src/accelerate/utils/__init__.py

```diff
@@ -110,6 +114,7 @@
         HfDeepSpeedConfig,
     )
 
+from .bnb import get_quantized_model
```

Comment:
```suggestion
from .bnb import quantize_model
```
Maybe `quantize_model` is a better name?

Reply:
Yes, I've changed it in this commit [0dcc440](https://github.com/huggingface/accelerate/pull/1626/commits/0dcc440d902228a57e7148198758eba6b1b09f45)